pkg/manifest/file.go

```diff
@@ -225,6 +228,13 @@ func (f *File) Write(path string) error {
 	return fp.Close()
 }
 
+// FilePath returns the original path of the file that was read, if any,
+// and a boolean indicating whether the file was read (true) or constructed
+// from env vars or other sources (false).
+func (f *File) FilePath() (string, bool) {
```

Comment:
Is this used? or was this something you added for future dev/debugging purposes?

Reply:
It was for handling a potential error that turned out to be impossible, ty!